modules/dag-l1/src/test/scala/org/tessellation/dag/l1/domain/block/BlockRelationsSuite.scala

```diff
@@ -0,0 +1,111 @@
+package org.tessellation.dag.l1.domain.block
+
+import cats.effect.{IO, Resource}
+
+import org.tessellation.dag.domain.block.DAGBlock
+import org.tessellation.dag.domain.block.generators._
+import org.tessellation.dag.l1.Main
+import org.tessellation.kryo.KryoSerializer
+import org.tessellation.schema.transaction
+import org.tessellation.sdk.sdkKryoRegistrar
+import org.tessellation.security.signature.Signed
+
+import weaver.MutableIOSuite
+import weaver.scalacheck.Checkers
+
+object BlockRelationsSuite extends MutableIOSuite with Checkers {
+
+  type Res = KryoSerializer[IO]
+
+  override def sharedResource: Resource[IO, BlockServiceSuite.Res] =
+    KryoSerializer.forAsync[IO](Main.kryoRegistrar ++ sdkKryoRegistrar)
+
+  test("blocks are not related") { implicit ks =>
+    forall { (block: Signed[DAGBlock], notRelatedBlock: Signed[DAGBlock]) =>
+      for {
+        hashedBlock <- block.toHashed[IO]
+        isRelated <- BlockRelations.areRelated(hashedBlock)
+        actual = isRelated(notRelatedBlock)
+      } yield expect.same(false, actual)
+    }
+  }
+
+  test("blocks are related - first block is parent of second") { implicit ks =>
```

Comment:
when the first block is parent of the second then blocks are related

Reply:
Changed.